charts/victoria-metrics-k8s-stack/values.yaml

```diff
@@ -11,7 +11,7 @@ argocdReleaseOverride: ""
 # -- also checkout here possible ENV variables to configure operator behaviour https://docs.victoriametrics.com/operator/vars.html
 victoria-metrics-operator:
   enabled: true
-  # -- Tells helm to remove CRD after chart remove
+  # -- Tells helm to clean up vm cr resources when uninstalling 
   cleanupCRD: true
```

Comment:
It looks, that cleanup CRD related variables must be moved into `victoria-metrics-operator.operator` section. 

For backward compatibility, we should support both section, templating it with _helpts.tpl.

Reply:
I don't see why move to `victoria-metrics-operator.operator` section, cleanCRD managed by victoria-metrics-operator chart, not by operator itself.